packages/miniflare/src/plugins/assets/index.ts

```diff
@@ -57,13 +57,17 @@ export const ASSETS_PLUGIN: Plugin<typeof AssetsOptionsSchema> = {
 		if (!options.assets) {
 			return [];
 		}
-		const assetsReverseMap = await createReverseMap(options.assets?.path);
 
 		const storageServiceName = `${ASSETS_PLUGIN_NAME}:storage`;
 		const storageService: Service = {
 			name: storageServiceName,
 			disk: { path: options.assets.path, writable: true },
 		};
+
+		const { encodedAssetManifest, assetsReverseMap } = await buildAssetManifest(
```

Comment:
what is the reasoning for combining the asset manifest and the reverse map under one fn? What are we optimizing for? Also, how does this play out with our plans of sharing more code between `workers-shared` and `wrangler`/`miniflare`. Does `buildAssetManifest` qualify in that category?

Reply:
Before it was traversing the asset directory twice, once each to create the reverse map and manifest, which wasn't great. And we can't really share buildAssetManifest because half of it happens in AUS for prod.

Reply:
yeah I thought as much, but wanted to sanity check